### ISSUE TITLE
Crashes when selecting any source in file mode

### ISSUE THREAD

**Issue by GoogleCodeExporter** [peteaky/android-xbmcremote#169]
```
* What steps will reproduce the problem?
1. Go to file mode for any media type (music/video)
2. Sources will be listed
3. Click a source to bring up its contents and the app will crash

* What version of XBMC Remote are you using?

Works fine in 0.6.3 crashes consistently in SVN from 486 - 499

* Which Android device are you using (and which ROM, if custom)?

HTC Hero CDMA, stock ROM
Software version: 1.56.651.2

* Which XBMC revision are you running and on on which platform?

Linux / r27017(ish?)  <-- don't recall if I've since updated and the
machine's offline

* Please provide any additional information below.

logcat attached
```

Original issue reported on code.google.com by `rsale...@gmail.com` on 21 Feb 2010 at 1:33

Attachments:
- [logcat-filemodecrash.txt](https://storage.googleapis.com/google-code-attachments/android-xbmcremote/issue-169/comment-0/logcat-filemodecrash.txt)

**Comment by GoogleCodeExporter** [comment]
```
see also:  http://groups.google.com/group/android-
developers/browse_thread/thread/ef1f206cfcab010d
```

Original comment by `phree...@gmail.com` on 21 Feb 2010 at 1:35
- Changed state: **Accepted**
- Added labels: **Priority-High**
- Removed labels: **Priority-Medium**, **Type-Enhancement**

**Comment by GoogleCodeExporter** [comment]
```
Issue 190 has been merged into this issue.
```

Original comment by `till.ess...@googlemail.com` on 21 Mar 2010 at 3:40

**Comment by GoogleCodeExporter** [comment]
```
hopefully fixed in r533
```

Original comment by `till.ess...@googlemail.com` on 21 Mar 2010 at 3:44

**Comment by GoogleCodeExporter** [comment]
```
Should be fixed in v0.6.7. Reopen if that's not the case.
```

Original comment by `phree...@gmail.com` on 31 Mar 2010 at 5:47
- Changed state: **Fixed**

**Comment by GoogleCodeExporter** [comment]
```
Very intermittent crashing on 0.6.5 and 0.6.6, but has been totally solid on 
0.6.7.

No longer reproducible on my end.  Works a treat!  Thanks everyone....
```

Original comment by `rsale...@gmail.com` on 3 Apr 2010 at 1:23
